ddtrace/tracer.py

```diff
@@ -1013,6 +1016,11 @@ def shutdown(self, timeout=None):
             for processor in chain(span_processors, SpanProcessor.__processors__):
                 if hasattr(processor, "shutdown"):
                     processor.shutdown(timeout)
+            if config._data_streams_enabled:
```

Comment:
Wouldn't we instead want to `try except` and then if `if config._data_streams_enabled:` log the warning?